mlir/test/Conversion/AffineToStandard/lower-affine.mlir

```diff
@@ -927,3 +927,12 @@ func.func @affine_parallel_with_reductions_i64(%arg0: memref<3x3xi64>, %arg1: me
 // CHECK:      scf.reduce.return %[[RES]] : i64
 // CHECK:    }
 // CHECK:  }
+
+#map_mod_8 = affine_map<(i) -> (i mod 8)>
+// CHECK-LABEL: func @affine_apply_mod_8
```

Comment:
Can you also check if using `... mod 1` doesn't lead to any unexpected behavior.